cmd/chain-watch/main.go

```diff
@@ -79,11 +82,23 @@ func chainWatch(provider *ethrpc.Provider, monitorOptions ethmonitor.Options) (*
 	// 	monitorOptions.PollingInterval = 5 * time.Millisecond
 	// }
 
+	snapshotFile := "/Users/peter/Dev/0xsequence/ethkit/cmd/chain-watch/snapshot.json"
```

Comment:
cmd/chain-watch/main.go command is an example of the bootstrapping mechanism.

to use... 

1. set ethkit-test.json and set a polygon url for the node
2. edit this hard-coded path..
3. run `go run .` from this directory
4. it will just run and keep writing to the snapshot file.. when you read enough blocks, just ctrl-c
5. when you `go run .` again, it will bootstrap from the file